bin/server/cmd/root.go

```diff
@@ -124,6 +129,7 @@ func init() {
 	rootCmd.PersistentFlags().BoolVar(&readonly, "readonly", false, "whether to run in read-only mode")
 	rootCmd.PersistentFlags().BoolVar(&demo, "demo", false, "whether to run using demo data")
 	rootCmd.PersistentFlags().BoolVar(&debug, "debug", false, "whether to enable debug level logging")
+	rootCmd.PersistentFlags().StringVar(&pgURL, "pg", "", "optional external PostgreSQL instance connection url; for example postgresql://user:secret@masterhost:5433/dbname?sslrootcert=cert")
```

Comment:
Since the parameter is already pg, how about omitting the postgresql:// part?

Reply:
I'd prefer follow the existing standard (PostgreSQL already defines its connection URIs pattern) instead of make some unnecessary changes - it may looks convenient at first but actually a dialect to learn.

Reply:
Sounds reasonable.